src/AccessibilityInsights.SharedUxTests/Settings/ConfigurationModelTests.cs

```diff
@@ -24,19 +24,14 @@ public class ConfigurationModelTests
         [ClassInitialize()]
         public static void ClassInit(TestContext context)
         {
-            var configDir = Path.Combine(context.TestRunDirectory, "config");
-            var userDir = Path.Combine(context.TestRunDirectory, "user");
+            string testBaseDirectory = Path.Combine(context.TestRunDirectory, context.TestName);
```

Comment:
`context.TestName` will be the name of the first test method executed in the class, which in this case happens to be `ConfigurationModelTest`. Is that expected? If not, could use `context.FullyQualifiedTestClassName` instead, though it's a bit verbose (here it would be `"AccessibilityInsights.SharedUxTests.Settings.ConfigurationModelTests"`)

Reply:
Good point. The entire path needs to not exceed MAX_PATH (defined as 260) characters. Using FullyQualifedTestClassName in this case means that we have about 127 characters after the root path of the repo, plus about 15 characters for file names. That means that we'll be find as long as the root repo path doesn't exceed 120 characters. That should be plenty, and we can use a hardcoded value if/when it becomes a problem. I didn't want to use a hardcoded value at first, since I wanted a pattern that we could copy elsewhere and it would "just work".

I've made the change to use FullyQualifiedTestClassName